File: assignment1_q1.py
x = input("Enter a positive integer n to find the sum of integers from 1 to n: ")
sum = 0
print("sum = ", end = '')
for i in range(1, int(x)+1):
    sum = sum + i
    if(i<int(x)):
        print(str(i) + " + ", end = '')
    else:
        print(i, end = '')
print(" = " + str(sum))

File: assignment1_q1.c
#include <stdio.h>

int main() {
	int i;
	int x;
	int sum = 0;

	printf("Enter a positive integer n to find the sum of integers from 1 to n: "); 
	scanf("%d", &x);

	printf("Sum = ");
	for (i = 1; i <= x; ++i){
		sum = sum + i;
		if(i<x){
			printf("%d + ", i);
		}
		else{
			printf("%d", i);
		}
	}
	printf(" = %d", sum);
	return 0;
}